users/src/main/scala/users/Main.scala

```diff
@@ -0,0 +1,27 @@
+package users
```

Comment:
* Do the source files need license headers?
* Packages, etc... should be under the `com.paidy.platform.exercises.xxx` namespace?

Reply:
nah; keep it simple, otherwise imports are crazy. It's not that this is production code,  or that we will publish this as a library.